scripts/tardis

```diff
@@ -36,6 +36,8 @@ parser.add_argument('--profile', action='store_true', help=
 parser.add_argument('--profiler_log_file', default='profiler.log', help=
 'name of the profiler output file')
 
+parser.add_argument('--gdb', action='store_true', help='print pid and pause')
```

Comment:
This is a debug statement I added specifically to enable debugging. If we don't want that capanility I can remove that commit but I think it's a handy option.

I wasn't able to start tardis with `gdb` to debug the C code, so I added it.